js/animations.js

```diff
@@ -0,0 +1,68 @@
+//Close the menu (shows up on md size) when clicked anywhere on the page
```

Comment:
Browser-side JS MUST be surrounded with an anonymous function call and be in explicit strict mode. i.e.:

```javascript
(function () {
  'use strict'

  // code here
}()
```

When you use jQuery, it should be:

```
window.jQuery(function ($) {
  'use strict'

  // code here
})
```

This gets around conflicts with `$` and only assumes a single global.